playbooks/inventory.yaml

```diff
@@ -0,0 +1,172 @@
+- name: Inventory
```

Comment:
# ToDo

Update the commit footers to state `#<issue number>` they relate to and when you do the docs commit(s) do the same, this time prefix with `closes #<issue number>`. this is in additional to having the PR number before the `close` parts

Reply:
Updated to include issue numbers in footer

Reply:
Dammit. How did I mess that up.

So the commit should be # <pull request number> # <issue number>??

Reply:
yes. see https://www.conventionalcommits.org/en/v1.0.0/ for full spec

Reply:
Updated in commit [f4e6b95](https://github.com/nofusscomputing/ansible_collection_centurion/pull/17/commits/f4e6b95609c506f6bcf91c8d0ebf32df2cdb58f9) and [6b0e50a](https://github.com/nofusscomputing/ansible_collection_centurion/pull/17/commits/6b0e50a5f075c82c2708f43eaa59e66cc24d5752)